cdo_local_uuid/__init__.py

```diff
@@ -78,6 +78,7 @@ def configure() -> None:
     env_base_dir_name = os.getenv("CDO_DEMO_NONRANDOM_UUID_BASE")
     if env_base_dir_name is None:
         return
+    # _logger.debug("env_base_dir_name = %r.", env_base_dir_name)
```

Comment:
Does this commented code need to still be here or was this for debugging?

Reply:
This was for debugging.

I've found it helpful to leave commented code like this as guides for what has helped figure out things in the past.  So I intentionally left the commented lines here, for those sessions where a runtime `--debug` doesn't cut it.

Reply:
To your question: it does not *need* to be here.